lib/mpd.nit

```diff
@@ -162,7 +162,7 @@ class MPDConnection
 		var time: nullable Int = null
 
 		socket.write("currentsong\n")
-		var rep = socket.read(1024)
+		var rep = socket.read(1024).to_s
```

Comment:
The original code was never updated I think, and if it's used, do not update with this PR.

Since EOF is not blocking and since it is reported when the connection is closed, we get the same behaviour with sockets as with stdin.

I'll change it in a future PR to have an adequate behaviour (typically wait for the first data to come in and do not block as long as something has been read).

Now, the million-dollar question, should we have stdin behave like this too ?